frontend/.coderabbit.yaml

```
@@ -0,0 +1,15 @@
+reviews:
+  profile: "assertive"
+  request_changes_workflow: false
+  high_level_summary: true
+  poem: true
+  review_status: true
+  collapse_walkthrough: false
+  auto_review:
+    enabled: true
+    drafts: false
+chat:
+  auto_reply: true
```

yamllint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)